packages/engine/Source/Renderer/Context.js

```diff
@@ -1340,7 +1340,14 @@ function continueDraw(context, drawCommand, shaderProgram, uniformMap) {
 
   if (defined(indexBuffer)) {
     offset = offset * indexBuffer.bytesPerIndex; // offset in vertices to offset in bytes
-    count = defaultValue(count, indexBuffer.numberOfIndices);
+    if (count) {
```

Comment:
Can we use the `defined` method here, following the [Coding Guide](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/CodingGuide#basic-code-construction)? The `defined` function (like `defaultValue`) checks for both `null` and `undefined`.